xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-notifiers/xwiki-platform-notifications-notifiers-api/src/main/java/org/xwiki/notifications/notifiers/internal/email/live/LiveNotificationEmailListener.java

```diff
@@ -44,10 +44,12 @@
  *
  * @since 9.6-RC1
```

Comment:
Since you are in this area, you might want to take that opportunity to fix this version (should be 9.6RC1)

Reply:
Ok will do that.